.github/workflows/bat.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: ['1.21', '1.22' ]
+        go-version: ['1.22.7', '1.23']
```

Comment:
Why specifically 1.22.7? Wouldn't it be better to have 1.22 so we track the latest version?